contents/docs/api/post-only-endpoints.mdx

```diff
@@ -99,6 +99,39 @@ curl -v -L --header "Content-Type: application/json" -d '{
 }' https://app.posthog.com/capture/
 ```
 
+### Group
+
+```bash
+curl -v -L --header "Content-Type: application/json" -d '{
+    "api_key": "<ph_project_api_key>",
+    "event": "$event",
+    "distinct_id": "1234",
+    "properties": {
+        "$groups": {"company": "<company_name>"}
```

Comment:
this is the example, when "company" is your group_type. One could use "organization" or anything else too, not sure if we want to call this out or not 🤷‍♀️